wca-regulations.md

```diff
@@ -213,11 +213,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 9b1a) Round format for these events is: "Average of 5". For a Combined Round, the round format is "Best of 2" for the first phase and "Average of 5" for the second phase.
     - 9b2) 6x6x6 Cube and 7x7x7 Cube.
         - 9b2a) Round format for these events is: "Mean of 3". For a Combined Round, the round format is "Best of 1" for the first phase and "Mean of 3" for the second phase.
-    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded, and 3x3x3 Multi-Blind.
+    - 9b3) 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded.
         - 9b3a) Round formats for these events are: "Best of X" (where X is 1, 2, or 3).
-        - 9b3b) For 3x3x3 Blindfolded, 4x4x4 Blindfolded, and 5x5x5 Blindfolded the WCA also recognizes "Mean of 3" rankings and records based on the times from "Best of 3" rounds.
-    - 9b4) 3x3x3 Fewest Moves.
-        - 9b4a) Round formats for this event are: "Best of X" (where X is 1 or 2) and "Mean of 3".
+        - 9b3b) For these events, the WCA also recognizes "Mean of 3" rankings and records based on the times from "Best of 3" rounds.
+    - 9b4) 3x3x3 Multi-Blind.
+        - 9b4a) Round format for this event is: "Best of X" (where X is 1, 2, or 3). For a Combined Round, the round format is "Best of X" (where X is 1 or 2) for the first phase and "Best of 3" for the second phase.
```

Comment:
This effectively prevent a Bo1/Bo2 combined round.
Not sure if it happened before, but since it is allowed under the current Regulations, and was also allowed with the previous 2019 draft.

Reply:
Seems Bo1/Bo2 was at least at Euro 2012, 2014 and 2016.

Reply:
Hmm, I've been staring at too many round formats to keep track. I've filed https://github.com/thewca/wca-regulations/issues/693 to find out exactly what we've held in the past.